rl_studio/envs/gazebo_envs.py

```diff
@@ -0,0 +1,260 @@
+import gym
+import rospy
+
+# import roslaunch
+import sys
+import os
+import signal
+from pathlib import Path
+
+from gazebo_msgs.msg import ModelState
+from gazebo_msgs.srv import SetModelState, GetModelState
+
+import subprocess
+from std_srvs.srv import Empty
+import random
+from rosgraph_msgs.msg import Clock
+
```

Comment:
what is the difference between this `gazebo_envs` and the same stored inside of `envs/f1/gazebo_envs`?, maybe we could merge into one. What do you think?

Reply:
Good idea. Thanks!